package.json

```diff
@@ -16,7 +16,6 @@
     "babel": "^5.6.14",
     "classnames": "^2.1.2",
     "flux": "^2.0.3",
-    "gulp-replace": "^0.5.3",
```

Comment:
Please restore! [It's still used](https://github.com/mesosphere/marathon-ui/pull/133/files#diff-b9e12334e9eafd8341a6107dd98510c9L183)